pkg/okteto/utils.go

```diff
@@ -0,0 +1,26 @@
+package okteto
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+// SetToken is used in test to setup token params, returns tempDir and error
+func SetToken(t *Token) (string, error) {
```

Comment:
If this is only used by tests I would have it in an internal package: https://golang.org/doc/go1.4#internalpackages.

I think having the function in a file like `pkg/internal/testutils/utils.go` would let you use the `SetToken` function in the places where it is being used and let others know that it is an utility only for testing purposes

Reply:
I've removed this, as with the new Context is no longer needed :)